sharding/src/main/scala/com/lightbend/akka/bench/sharding/latency/LatencyBenchEntity.scala

```diff
@@ -90,16 +95,30 @@ class LatencyBenchEntity extends Actor with ActorLogging { // FIXME: FIX CASSAND
       // simple roundtrip
       sender() ! PongSecond(msg)
 
-//    case msg: PersistAndPing =>
-//      // roundtrip with write
-//      log.debug("Got persist-ping from [{}]", sender())
-//      val before = System.nanoTime()
-//      persist(PingObserved(msg.sentTimestamp)) { _ =>
-//        val persistMs = (System.nanoTime() - before).nanos.toMillis
-//        PersistenceHistograms.persistTiming.recordValue(persistMs)
-//        persistentPingCounter += 1
-//        sender() ! Pong(msg)
-//      }
+    case msg: PersistAndPing =>
+      // roundtrip with write
+      val before = System.nanoTime()
+      persist(PingObserved(msg.pingCreatedNanoTime)) { _ =>
+        val persistMs = (System.nanoTime() - before).nanos.toMillis
+        PersistenceHistograms.singlePersistTiming.recordValue(persistMs) // FIXME this is concurrent access, wrong!
+        persistentPingCounter += 1
+      }
+      persistAll(List.fill(99)(PingObserved(msg.pingCreatedNanoTime))) { _ =>
+        persistentPingCounter += 99
+        sender() ! PongFirst(msg, started)
+        context.stop(self)
+      }
+      
+    case msg: PersistPingSecond =>
+      sender() ! RecoveredWithin((recovered - started).nanos.toMillis, persistentPingCounter)
+      deleteMessages(lastSequenceNr)
```

Comment:
yes, but we know that C* doesn't like deletes and we recommend against them.
if I understand the test it would have to run very long to grow much since there are so many actors